NetCasbin/Model/Assertion.cs

```diff
@@ -154,6 +153,12 @@ internal bool TryAddPolicy(List<string> rule)
             {
                 return false;
             }
+
+            if (TryGetPriorityIndex(out int index))
```

Comment:
Use the same name as Go and Java, why add "TryXXX"?

Reply:
This is not a Public API. `TryXXX` can safe-return without exception, and can as an atomic operation instead of check and get.
According to golang casbin,  The public API is only `SortPoliciesByPriority` and I used the same name.